apps/user-office-backend/src/queries/AdminQueries.ts

```diff
@@ -94,17 +95,60 @@ export default class AdminQueries {
         ];
 
       const proto = Object.getPrototypeOf(element);
-      const names = Object.getOwnPropertyNames(proto).filter(
-        (item) => item !== 'constructor'
-      );
+      if (!proto.constructor.name.startsWith('Admin')) {
+        const names = Object.getOwnPropertyNames(proto).filter(
+          (item) => item !== 'constructor'
+        );
 
-      const classNamesWithMethod = names.map(
-        (item) => `${proto.constructor.name}.${item}`
-      );
+        const classNamesWithMethod = names.map(
```

Comment:
Constructing the classNameWithMethod could be moved out to a function declaration to keep things DRY